common/test/acceptance/tests/studio/test_studio_outline.py

```diff
@@ -198,71 +204,128 @@ def _build_fixture(self, unit_state):
             subsection if unit_state.publish_state == self.PublishState.NEVER_PUBLISHED
             else subsection.add_children(
                 XBlockFixtureDesc('vertical', name, metadata={
-                    'visible_to_staff_only': True if unit_state.is_locked else None
+                    'visible_to_staff_only': True if unit_state.is_locked else None,
+                    'group_access': {50: [1]} if unit_state.is_restricted else {},
```

Comment:
Currently we have the javascript tests for the expected form body and verifying that the divided warning messages are shown, we also have bok choy tests for similar functionality that probably doesn't need to be duplicated. What we can add are bok-choy tests that will go through a user's workflow of creating a group configuration, and then setting the group access on a unit via the course outline page